bin/bechamel-html/dune

```diff
@@ -0,0 +1,15 @@
+(executable
+ (name bechamel_html)
+ (modules bechamel_html html_file)
+ (package bechamel)
+ (public_name bechamel-html))
+
+(rule
+ (targets html_file.ml)
+ (deps (:src ../../html/index.html))
```

Comment:
you don't need to declare this as a dependency - it will be inferred automatically. Actually, as we discussed in another repo, this is one case where you can use the short form and infer `(targets)` as well! Lean and clean!

Reply:
It looks great thanks ! Otherwise, what do you think of how I generate the ml file ?

Reply:
you don't even need `(cat %{dep:file.ml})`, `(cat file.ml)` will do the trick. As for how the ml file is generated, this solution works so it's all good. If you're willing to pull some dependencies, you can use `ppx_blob` or `crunch` to do that in a more integrated way, but if this works, it works.

Reply:
Ok to depend only on `cat`.